app/src/main/java/com/github/se/cyrcle/model/parking/ParkingViewModel.kt

```diff
@@ -27,7 +27,7 @@ const val DEFAULT_RADIUS = 100.0
 const val MAX_RADIUS = 1000.0
 const val RADIUS_INCREMENT = 100.0
 const val MIN_NB_PARKINGS = 10
-const val NB_REPORTS_THRESH = 10
+const val NB_REPORTS_THRESH = 1
```

Comment:
It's good value for now. Well done for putting the constants it now comes in handy :+1: